src/components/FileRepoSidebar.js

```diff
@@ -147,46 +123,184 @@ const FileRepoSidebar = ({ projectId, index, style, sqon, effects, ...props }) =
               >
                 <PillInputWithButton
                   options={{
-                    'Clinical (Participant)': clinicalDataParticipants({
+                    'Participant only': fileManifestParticipantsOnly({
                       sqon,
                       columns: state.columns,
                     }),
-                    'Clinical (Family)': clinicalDataFamily({ sqon, columns: state.columns }),
+
+                    'Participant and family': () => {
+                      return effects.setModal({
+                        title: 'Download Manifest (Participant and Family)',
+                        component: (
+                          <FamilyManifestModal
+                            sqon={sqon}
+                            index={index}
+                            projectId={projectId}
+                            columns={state.columns}
+                          />
+                        ),
+                      });
+                    },
                   }}
                   render={({ loading }) => {
                     return (
                       <React.Fragment>
-                        <DownloadIcon loading={loading} />
-                        DOWNLOAD
+                        <DownloadIcon loading={loading} />DOWNLOAD
                       </React.Fragment>
                     );
                   }}
                 />
               </div>
-              <LoadingOnClick
-                onClick={downloadBiospecimen({ sqon, columns: state.columns })}
-                render={({ onClick, loading, disabled }) => (
-                  <Button
-                    css={`
-                      flex-grow: 1;
-                      padding-left: 15px;
-                    `}
-                    disabled={disabled}
-                    onClick={onClick}
-                  >
-                    <DownloadIcon loading={loading} />BIOSPECIMEN
-                  </Button>
-                )}
-              />
-            </div>
-          );
-        }}
-      />
+            );
+          }}
+        />
+        <ColumnsState
+          projectId={projectId}
+          graphqlField="file"
+          render={({ state }) => {
+            return (
+              <div>
+                <Heading style={{ color: '#343434', fontSize: 14, marginBottom: 5 }}>
+                  Selected File
+                </Heading>
+                <div
+                  css={`
+                    display: flex;
+                    margin-bottom: 13px;
+                  `}
+                >
+                  <LoadingOnClick
+                    onClick={async () => {
```

Comment:
`async` but no `await`. this can be very confusing for readers. you should either pick using `async`  with `await` and don't call `.then` or don't bother with `async` at all. I know that sometimes you'll see our code mix them, but it's in places where it's much easier to reason about why it's done like that.

example:

```
let response = await fetch(url).then(r => r.json())
```
it's mixed there, but it's because it's easy to follow.
the same can, and often is written as

```
let response = await fetch(url)
let data = await response.json()
```

which I think is totally fine. my suggestion for you is to choose one style to get more familiarized with it